denizen_scripts/hub1/repo-link/global_data_handler.dsc

```diff
@@ -24,20 +24,24 @@ global_data_handler:
 
         - yaml id:<[GlobalYaml]> create
         - yaml id:<[GlobalYaml]> savefile:<[Directory]>
-
+    # % ██ [ Track Player ] ██
+      - yaml id:data_handler set network.names:->:<[Name]>
+      - yaml id:data_handler set network.uuids:->:<[UUID]>
     on bungee player switches to server:
     # % ██ [ Cache Player Info ] ██
       - define Name <context.name>
       - define UUID <context.uuid>
       - define Server <context.server>
+      - define Old_Server <yaml[data_handler].read[players.<[UUID]>.server]>
```

Comment:
This will be invalid if the player just joins the network and isn't switching from another server
```suggestion
      - if <yaml[data_handler].contains[players.<[UUID]>.server]>:
        - define Old_Server <yaml[data_handler].read[players.<[UUID]>.server]>
```